src/PresentationalComponents/Notifications/TabsMenu.tsx

```diff
@@ -10,19 +10,18 @@ import {
   Menu,
   MenuContent,
   MenuGroup,
-  MenuInput,
-  MenuItem,
   MenuList,
   TextInput,
   Title,
 } from '@patternfly/react-core';
+import { MenuItem, MenuSearch, MenuSearchInput } from '@patternfly/react-core/dist/dynamic/components/Menu';
 import { SearchIcon } from '@patternfly/react-icons';
 import { useLocation, useNavigate } from 'react-router-dom';
-import PropTypes from 'prop-types';
+import PropTypes, { string } from 'prop-types';
```

Comment:
```suggestion
```
this doesn't seem to be used

Reply:
ohh yeah forgot to take this one out. Thanks!